pkg/ui/workspaces/cluster-ui/src/store/clusterSettings/clusterSettings.reducer.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { DOMAIN_NAME } from "../utils";
+import {
+  SettingsRequestMessage,
+  SettingsResponseMessage,
+} from "../../api/clusterSettingsApi";
+
+export type ClusterSettingsState = {
```

Comment:
Can you use RequestState here.

Reply:
@xinhaoz are you backporting all the requestState changes? you might need if we want to backport this using it

Reply:
Ah if this is to be backported then we can leave as is for now.